help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,180 @@
+
+  ![Destination Detail](../../assets/catalog/advertising/acxiom/image-destination-details.png)
+
+* **Name (Required)** - The name the destination will be saved under
+* **Description** -  Short explanation of the destination's purpose
+* **Bucket Name (Required)** - Name of the S3 bucket set up on S3
+* **Folder Path (Required)** - If subdirectories in a bucket are used a path must be defined, or '/' to reference the root path.
+* **File Type** - Currently the only file type Acxiom processing will be expecting is CSV
+* **CSV Options**
+  * **Delimiter** - Field delimiter used for the file.  Options include:
+    * **Pipe** (|)
+    * **Semicolon** (;)
+    * **Tab** (\t)
+    * **Colon** (:)
+    * **Comma** (,) (Default)
+  * **Quote Character** - Includes the selected quote value around all columns in the resulting file.  Options include:
+    * **Null Character** (\u0000) (Default)
+    * **Double Quotes** (")
+  * **Escape Character** - When fields include characters that may interfere with the delimiters or quotes such as double quotes or commas, an escape character would need to be provided so they can be properly evaluated during import.  This value indicates what that escape character should be.  Values include:
+    * **Single Quote** (')
+    * **Back Slach** (\) (Default)
+  * **Empty Value** - If the value does not exist on the source this indicates the value that should stand in for that missing value.  Options include:
+    * **Empty String** () (Default)
+    * **Null** (null)
+    * **Empty String in Double Quotes** ("")
+  * **Null Value** - If the value does exist, but the value is a null value this indicate how that should be represented on the file.  Options include:
+    * **Empty String** ()
+    * **Null** (null) (Default)
+    * **Empty String in Double Quotes** ("")
+  * **Compression Format** - Specifies if the resulting file will be compressed or not.  Current option include GZIP, and None
+  * **Include manifest file** -  This Boolean option appears only when the GZIP option is selected.  When selected a file outlining details on the file included in the zip file is provided.
+<br>  ![CSV Options](../../assets/catalog/advertising/acxiom/image-destination-csv-options.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Data Governance Policy and Enforcement Action ##
+Option to select data governance policy  [data-governance overview](../../../data-governance/home.md)
+<br>  ![Data Governance policy](../../assets/catalog/advertising/acxiom/image-destination-governance.png)
+
+
```

Comment:
there is no need for the sections between lines 121 - 168. all these steps of the workflow are explained in the page that we are linking to further above. (see my comment about adding section Activate audiences to this destination {#activate})

Reply:
Dropped these lines in the document